tests/acceptance/features/apiLocks/unlockFiles.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Unlock locked items
+  As a user
+  I want to unlock the resources previously locked by myself
+  So that other users can make changes to the resources
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+
+  @issue-7696
+  Scenario Outline: unlock a file in project space
+    Given using spaces DAV path
+    Given the administrator has assigned the role "Space Admin" to user "Alice" using the Graph API
+    And user "Alice" has created a space "project-space" with the default quota using the Graph API
+    And user "Alice" has uploaded a file inside space "project-space" with content "some data" to "textfile.txt"
+    And user "Alice" has locked file "textfile.txt" inside space "project-space" setting the following properties
+      | lockscope | <lock-scope> |
+    When user "Alice" unlocks the last created lock of file "textfile.txt" inside space "project-space" using the WebDAV API
```

Comment:
do we already have coverage for locking project space file?

Reply:
I try to search bu could not find so I think there is no coverage for locking and unlocking project spaces

Reply:
Then I would suggest adding locking and unlocking tests

Reply:
there is some coverage for locking the project space file https://github.com/owncloud/ocis/blob/841ab73af20a70d0723eb5d67de65cbaab223afb/tests/acceptance/features/apiLocks/lockFiles.feature#L91

For the first batch of lock/unlock tests IMO we can shift the tests from core to ocis and provide the coverage for unlock as per there is coverage for locking and other edge cases can be added later.